hydra_base/lib/template.py

```diff
@@ -2023,10 +2106,6 @@ def get_network_as_xml_template(network_id,**kwargs):
             resource_name   = etree.SubElement(group_resource, "name")
             resource_name.text   = group_i.group_name
 
-           # layout = _get_layout_as_etree(group_i.layout)
```

Comment:
Why is this commented out? This will stop group layouts from being exported.

Reply:
This was already commented . it has been removed. it is not in the code.